tests/unit/aiplatform/test_training_jobs.py

```diff
@@ -387,6 +392,13 @@ def mock_python_package_to_gcs(self):
     def mock_dataset(self):
         ds = mock.MagicMock(datasets.Dataset)
         ds.name = _TEST_DATASET_NAME
+        ds._gca_resource = GapicDataset(
```

Comment:
Added this as the client SDK tests for `Dataset._gca_resource.metadata_schema_uri`